src/test/CMakeLists.txt

```diff
@@ -507,9 +507,12 @@ add_dependencies(check
   cython_modules)
 
 add_ceph_test(test-ceph-helpers.sh ${CMAKE_CURRENT_SOURCE_DIR}/test-ceph-helpers.sh)
+add_ceph_test(ceph_objectstore_tool.py ${CMAKE_CURRENT_SOURCE_DIR}/ceph_objectstore_tool.py)
+set_property(TEST test-ceph-helpers.sh ceph_objectstore_tool.py
+  PROPERTY LABELS Racing LongRunning)
```

Comment:
@alimaredia they are documented in the commit message =) , see 19a8b5cb58fd13782bb749dfa942ae41e6443ce2

just random strings, so we can select the racing tests more easily with ctest.